server/middlewares/validators/videos/video-imports.ts

```diff
@@ -38,13 +42,17 @@ const videoImportAddValidator = getCommonVideoEditAttributes().concat([
     .custom(isVideoNameValid).withMessage(
       `Should have a video name between ${CONSTRAINTS_FIELDS.VIDEOS.NAME.min} and ${CONSTRAINTS_FIELDS.VIDEOS.NAME.max} characters long`
     ),
+  body('videoPasswords')
```

Comment:
I think you can extract some `isValidPasswordProtectedPrivacy` logic here: if `videoPasswords` is set, at least check it's a unique array of valid passwords.
Then you can create another function that checks password consistency (video is password protected so `body.videoPasswords` must exist